backend/functions/lib/adapters/controllers/case.assignment.controller.test.ts

```diff
@@ -1,6 +1,7 @@
 import { CaseAssignmentController } from './case.assignment.controller';
-import { AttorneyAssignmentResponseInterface } from '../types/case.assignment';
 import { applicationContextCreator } from '../utils/application-context-creator';
+import { THROW_PERMISSIONS_ERROR_CASE_ID } from '../../cosmos-humble-objects/fake.cosmos-client-humble';
+
 const functionContext = require('azure-function-context-mock');
 
 describe('Case Assignment Creation Tests', () => {
```

Comment:
I thought for certain that we replaced this applicationContextCreator() here with the mockApplicationContext thing.  But apparently we didn't.  Maybe we did that somewhere else?  We should make that change here.

Reply:
My local history doesn't reflect me ever having it changed. I'm not opposed to changing it, but do we want to just get done and change this later?

Reply:
sure

Reply:
Revisit this later.